AT_EFD/docker-compose.yaml

```diff
@@ -1,103 +1,154 @@
-version: "3"
+version: '3.6'
 
 services:
-        EFD_Updater:
-                image: lsstts/efd_writers
-                volumes:
-                        - ./efd_population.sh:/home/saluser/efd_population.sh
-                network_mode: "host"
-                entrypoint: ["/home/saluser/efd_population.sh"]
+#        EFD_Updater:
+#                image: lsstts/efd_writers
+#                volumes:
+#                        - ./efd_population.sh:/home/saluser/efd_population.sh
+#                network_mode: "host"
+#                entrypoint: ["/home/saluser/efd_population.sh"]
 
         EFD_MariaDB:
                 image: mariadb:latest
-                ports: 
-                        - "3306:3306"
                 environment:
                         - MYSQL_ROOT_PASSWORD=lssttest
                         - MYSQL_DATABASE=EFD
                         - MYSQL_USER=efduser
                         - MYSQL_PASSWORD=lssttest
-                restart: on-failure
-                network_mode: "host"
-                #volumes:
-                        #- ${PWD}/mariadb:/var/lib/mysql
+                volumes:
+                        - ${HOME}/efd/mysql:/var/lib/mysql
+                networks:
+                        control-network:
+                                ipv4_address: "192.168.1.2"
+
         EFD_ATDome:
-                image: lsstts/efd_writers
+                image: lsstts/efd_writers:DM-20529
```

Comment:
a tag version ?

Reply:
Yes, I'm still working on getting a tag together for this.